straxen/plugins/events/corrected_areas.py

```diff
@@ -267,7 +268,13 @@ def compute(self, events):
             # cS2 bottom should be corrected by photon ionization, but not cS2 top
             cs2_top_wo_elifecorr = cs2_top_wo_picorr
             cs2_bottom_wo_elifecorr = cs2_bottom_wo_picorr * self.cs2_bottom_top_ratio_correction
-            result[f"{peak_type}cs2_wo_elifecorr"] = cs2_top_wo_elifecorr + cs2_bottom_wo_elifecorr
+            cs2_wo_elifecorr = cs2_top_wo_elifecorr + cs2_bottom_wo_elifecorr
+            # scale top and bottom to ensure total cS2 is conserved, since the time
+            # dependence of it has been already corrected by SEG correction
+            cs2_top_wo_elifecorr *= cs2_wo_picorr / cs2_wo_elifecorr
+            cs2_bottom_wo_elifecorr *= cs2_wo_picorr / cs2_wo_elifecorr
+            cs2_wo_elifecorr = cs2_wo_picorr
```

Comment:
Sorry guys, I am slow: is `cs2_top_wo_elifecorr + cs2_bottom_wo_elifecorr == cs2_wo_elifecorr` ? If the scaling is correctly done, it should be the case, right? At least this is what I got from the calculation I showed in the [slack](https://xenonnt.slack.com/archives/C049JJZ2U3A/p1703063137682539?thread_ts=1702997110.354889&cid=C049JJZ2U3A)

Reply:
Yes, this might be clear to illustrate what happens.
```
t = t * cs2 / (t+b)
b = b * cs2 / (t+b)
cs2 = cs2
```

Reply:
Thank you both, I am happy now!